packages/atomic/src/index.html

```diff
@@ -52,23 +52,35 @@
         margin: 0;
       }
 
-      .interface-container {
-        width: 100%;
-        max-width: 1250px;
-        margin: 0 auto;
-        display: grid;
-        grid-template-columns: 25% 1fr;
+      atomic-search-interface {
+        grid-template-columns: 10% 1fr 3fr 10%;
+        grid-template-rows: 22px 50px 22px 1fr;
         gap: 20px;
+        display: grid;
+        margin: 0 auto;
       }
 
-      .search-box-container {
+      .header {
         background-color: var(--atomic-background-variant);
-        padding: 22px 50px;
+        grid-column: 1 / span 4;
+        grid-row: 1 / span 3;
       }
 
-      .footer-item {
-        grid-column-start: 2;
-        grid-row-start: 3;
+      atomic-search-box {
+        grid-column: 3 / auto;
+        grid-row: 2 / auto;
+        max-width: 1250px;
```

Comment:
I still find this huge for a searchbox, using the mouse to reach the button is a bit painful when this is the case, in the mockups the max width is 650px which I find reasonable